### ISSUE TITLE
AttributeError when training

### ISSUE THREAD

**Issue by fabiovii** [ildoonet/tf-pose-estimation#101]
When issuing the following command to train the network as the README explains

`python3 src/train.py --model=cmu --datapath={datapath} --batchsize=64 --lr=0.001 --modelpath={path-to-save}`

I get the following error
```
File "src/train.py", line 197, in <module>
    enqueuer.start()
File "tf-pose-estimation/src/pose_dataset.py", line 420, in start
    super(threading.Thread).start()
AttributeError: 'super' object has no attribute 'start'
```

I changed the line 420 into 
`super().start()`
and now the code runs fine.

Did you have the same problem?

**Comment by scyoyo** [comment]
Yes I had it. But the code in pose_dataset.py should be `super(DataFlowToQueue).start()`?
`super().start()` reports error. I fixed it by replacing line 420 with `threading.Thread.start(self)`

**Comment by fabiovii** [comment]
> But the code in pose_dataset.py should be super(DataFlowToQueue).start()?
super().start() reports error

In the _master_ branch the code is `super(DataFlowToQueue).start()` and it does not run on my machine. On the _devel_ branch it is just `super().start()`

> I fixed it by replacing line 420 with threading.Thread.start(self)

With this modification I get the same behaviour as using `super().start()`, in the sense that the training procedure seems to run correctly.

What line 420 should be?
